pkg/validators/test_am0007.go

```diff
@@ -0,0 +1,29 @@
+package validators
+
+import (
+	"github.com/mt-sre/addon-metadata-operator/internal/testutils"
+	"github.com/mt-sre/addon-metadata-operator/pkg/utils"
+)
+
+func init() {
+	TestRegistry.Add(TestAM0007{})
+}
+
+type TestAM0007 struct{}
+
+func (v TestAM0007) Name() string {
+	return AM0007.Name
+}
+
+func (v TestAM0007) Run(mb utils.MetaBundle) (bool, string, error) {
+	return AM0007.Runner(mb)
+}
+
+func (v TestAM0007) SucceedingCandidates() []utils.MetaBundle {
+	return testutils.DefaultSucceedingCandidates()
+}
+
+// not implemented
+func (v TestAM0007) FailingCandidates() []utils.MetaBundle {
+	return []utils.MetaBundle{}
+}
```

Comment:
Let's add some failing candidates here?

Reply:
This is tricky, as we would need to package failing bundles, and we don't have an easy way to do this right now.

Reply:
Ahh makes sense. I just looked into it. We'd have to 
- define a static YAML/JSON of a CSV 
- unmarshal it into an `unstructured.Unstructured` object 
- feed that object to `registry.NewBundle(.., ..., obj)`
- And finally, build a failing `[]utils.MetaBundle` out of it.

I can wrap it up in a separate PR :)
This PR looks good to me :)